specs/network_details_for_hardware_inventory.adoc

```diff
@@ -0,0 +1,138 @@
+= Specification for adding network details in node hardware inventory
+
+Add network details also as a part of node hardware inventory
+
+== Problem description
+
+Hardware inventory details are used to show exact status of the particular
+node. Currently network details are missing in node hardware inventory.
+we have to add network details like IPv4, netmask, status as a part of
```

Comment:
I expect that Netifaces library supports it, but I would like to ask if you count also with IPv6 support?